README.md

```diff
@@ -132,6 +165,41 @@ Note: as with many sensitive operations, taking different actions when an
 account exists vs. when an account doesn't exist can lead to a information
 disclosure through timing attacks.
 
+
+#### Verifying
+
+Similar to the approach above, there is also a convenience method for verifying
+the current user.
+
+```ruby
+class SessionsController < ApplicationController
+  include ThisData::TrackRequest
+
+  def create
+    if User.authenticate(params[:email], params[:password])
+
+      # They used the right credentials, but does this login look unusual?
+      response = thisdata_verify
+
+      if ThisData::RISK_LEVEL_GREEN.eql? response["risk_level"]
```

Comment:
Yeah that's a habit I have of calling methods on things I know exist. In this case either will work, and your one is more grokable